### DISCUSSION TITLE
v0.12-dev

### DISCUSSION THREAD

**Discussion by sriramveeraghanta** [makeplane/plane#2120] [Announcements]
We have just published our [August updates](https://plane.so/blog/august-round-up-plane-deploy-member-profile-new-rte-export-data-and-more), and a day later, we are excited to introduce our latest release, **Plane v0.12-dev**. This release is packed with power, taking [Plane Deploy](https://docs.plane.so/deploy) to a whole new level. But that's not all; there's more to it, including time zone enhancements, improved Gantt charts, table support for our issue detail-rich text editor, and much more. Dig in to learn more.

### 🌠 Face-lift to Plane Deploy

We’ve made some significant upgrades to Plane Deploy, with the goal of making customer feedback process smoother and faster. Here are the upgrades ⬇️

- **Commenting on Public Boards**: When your boards are made public with commenting enabled, external users can start adding comments to the issues.
    
    > Note that, users need to sign in on the boards in order to comment. Also, when these users add comments they’ll not be added as the project members but instead added as contributors to your project.
    > 
- **Upvotes and Downvotes**: Contributors who have logged in to your public board can also upvote or downvote issues to help prioritize tasks within your project.
- **View Issues with Peek-Overview**: Users or contributors can now view issue details on your public boards using a "peek-overview" mode. Please note that not all information will be visible. The attributes that can be seen are as follows: Title, Description, Priority, Due Date, Public Comments, Upvotes/Downvotes, and Reactions.
- **Reactions**: Just like upvotes, users can react with emoji’s.

![plane-deploy-min](https://github.com/makeplane/plane/assets/9498163/c0344f54-47b1-4d10-86ad-9113223e51a6)

([#2006](https://github.com/makeplane/plane/pull/2006), [#2010](https://github.com/makeplane/plane/pull/2010), [#2012](https://github.com/makeplane/plane/pull/2012), [#2013](https://github.com/makeplane/plane/pull/2013), [#2017](https://github.com/makeplane/plane/pull/2017), [#2022](https://github.com/makeplane/plane/pull/2022), [#2026](https://github.com/makeplane/plane/pull/2026), [#2030](https://github.com/makeplane/plane/pull/2030), [#2039](https://github.com/makeplane/plane/pull/2039), [#2050](https://github.com/makeplane/plane/pull/2050), [#2051](https://github.com/makeplane/plane/pull/2051), [#2054](https://github.com/makeplane/plane/pull/2054), [#2057](https://github.com/makeplane/plane/pull/2057), [#2058](https://github.com/makeplane/plane/pull/2058), [#2068](https://github.com/makeplane/plane/pull/2068), [#2069](https://github.com/makeplane/plane/pull/2069), [#2073](https://github.com/makeplane/plane/pull/2073), [#2077](https://github.com/makeplane/plane/pull/2077), [#2081](https://github.com/makeplane/plane/pull/2081), [#2084](https://github.com/makeplane/plane/pull/2084), [#2096](https://github.com/makeplane/plane/pull/2096), [#2099](https://github.com/makeplane/plane/pull/2099), [#2100](https://github.com/makeplane/plane/pull/2100), [#2108](https://github.com/makeplane/plane/pull/2108), [#2109](https://github.com/makeplane/plane/pull/2109), [#2110](https://github.com/makeplane/plane/pull/2110), [#2116](https://github.com/makeplane/plane/pull/2116)**)**

### 🎉 **Gantt comes out of Beta**

Ten commits, and thousand lines later, Gantt layout now comes out of Beta with killer features. 

- View issues in Gantt layout, smoother faster and accurate—renders blocks with start date and due-date.
- Complete revamp of the design, making the gantt layout more pleasing and easy to work with.
- A new sidebar to spot issues quickly.
- Left and right navigators to jump to an issue that’s out of timeline.
- Move gantt blocks horizontally to adjust start date and due-date of an issue.
- Supports filters from View options.

![gantt](https://github.com/makeplane/plane/assets/9498163/bc03fcaa-a5f7-476b-af5d-7f5be9a29cae)

([#1900](https://github.com/makeplane/plane/pull/1900), [#2024](https://github.com/makeplane/plane/pull/2024), [#2070](https://github.com/makeplane/plane/pull/2070), [#2074](https://github.com/makeplane/plane/pull/2074), [#2101](https://github.com/makeplane/plane/pull/2101))

### 👀 **Peek overviews for Spreadsheet layout**

We've introduced a new feature, **Peek Overview**, which provides a quick look at your issues. This view is compatible with the spreadsheet layout and is also available for deployed public projects.

- Occupies half of your screen, providing you with a view of activity related to other issues.
- Allows users to edit issues directly within the Peek Overview, eliminating the need to navigate through multiple pages.
- Fast and highly responsive.

![PeakOverview](https://github.com/makeplane/plane/assets/9498163/3d9dddec-6820-44fe-9ed5-a584ee109094)

([#1979](https://github.com/makeplane/plane/pull/1979), [#2021](https://github.com/makeplane/plane/pull/2021), [#2043](https://github.com/makeplane/plane/pull/2043), [#2064](https://github.com/makeplane/plane/pull/2064), [#2075](https://github.com/makeplane/plane/pull/2075))

### **🕝 Better comms with Timezones**

With the integration of time zones, all activities on Planes are now more accurately recorded. Update your time zone settings in your user profile to enhance communication with your project members.

- Local time can be displayed based on the time zone set in user profiles.
- Activities related to issues are logged according to the time zone specified by the user.

![timezones-min](https://github.com/makeplane/plane/assets/9498163/9b75f937-b4d7-4c1a-a8af-24348df2877a)

([#2002](https://github.com/makeplane/plane/issues/2002), [#2009](https://github.com/makeplane/plane/issues/2009), [#2089](https://github.com/makeplane/plane/pull/2089)**)**

### 🚀 New attribute to Issues: start date

You can now add a start date to issues, facilitating better planning and estimation. This addition also aids in filtering and sorting issues.

- Helps view your issues based on timelines on the gantt layout.
- Filter or order issues based on the `start-date` attribute.
- [coming-soon] notifications for issues based on start-date.

![start-date](https://github.com/makeplane/plane/assets/9498163/0990dbc0-8da5-408a-b66e-4489b0eecd77)

**([#1813](https://github.com/makeplane/plane/pull/1813),** [#1955](https://github.com/makeplane/plane/pull/1955), [#2007](https://github.com/makeplane/plane/pull/2007))

### 😉 **Tables on the Text**

In our previous release, we unveiled a comprehensive revamp of our Rich Text Editor for issue details. Expanding on that, we now offer the ability to include tables within those details.

- Generate tables using slash commands.
- Add or remove rows and columns with ease.
- Supports adding headers for better column identification.

![Table_Tip_tap](https://github.com/makeplane/plane/assets/9498163/919cca94-c3c0-409f-bc49-a555b82e480f)

([#2008](https://github.com/makeplane/plane/pull/2008), [#2033](https://github.com/makeplane/plane/pull/2033), [#2061](https://github.com/makeplane/plane/pull/2061), [#2092](https://github.com/makeplane/plane/pull/2092))

### 📝 **`Mark as read` on Notifications**

Too many notifications? Hit `Mark as read` from the notifications dropdown to declutter all your notifications.

![Mark_all_as_read](https://github.com/makeplane/plane/assets/9498163/bb30cf58-ee77-468a-aba0-7fbbd58c0629)

 ([#1963](https://github.com/makeplane/plane/pull/1963), [#1982](https://github.com/makeplane/plane/pull/1982))

### 🏗️  Improvements

- Now, Guests and Viewers can easily leave a project anytime. Simply hover the project in the sidebar click on the 'Leave' option to fulfil your request. ([#2042](https://github.com/makeplane/plane/pull/2042), [#2079](https://github.com/makeplane/plane/pull/2079), [#2083](https://github.com/makeplane/plane/pull/2083))
- Added the "Group by Assignees" filter for issues. ([#1957](https://github.com/makeplane/plane/pull/1957))
- Added the issue activity redirection to cycles and modules. ([#1973](https://github.com/makeplane/plane/pull/1973))
- Updated the user activity endpoint to return only workspace activities. ([#1980](https://github.com/makeplane/plane/pull/1980))
- Improved development workflows by incorporating PR build updates, while also eliminating image updates with every merge. ([#1985](https://github.com/makeplane/plane/pull/1985))
- Updated Docker files for Plane-Web and Plane-Deploy, culminating in successful builds. ([#1987](https://github.com/makeplane/plane/pull/198))
- Implemented new module status icons and refreshed the color palette for status indicators. ([#2011](https://github.com/makeplane/plane/pull/2011))
- Improved issue detail sidebar. ([#2014](https://github.com/makeplane/plane/pull/2014))
- Improved the themes in the Kanban Boards and Sign In pages. ([#2055](https://github.com/makeplane/plane/pull/2055))

### 🐛  Bug Fixes

- Proper date validation has been added to cycles and modules. ([#1945](https://github.com/makeplane/plane/pull/1945))
- The project creation response now includes member roles and member statuses. ([#1962](https://github.com/makeplane/plane/pull/1962))
- The problem with mutation issues in workspace member invitations has been resolved. ([#1966](https://github.com/makeplane/plane/pull/1966))
- The mutation issues in project member invitations has also been fixed. ([#1967](https://github.com/makeplane/plane/pull/1967))
- The `Tiptap` editor's maximum width has been adjusted to occupy the proper screen size. ([#1968](https://github.com/makeplane/plane/pull/1968))
- After deleting a project, users will now be redirected to the projects list page. ([#1970](https://github.com/makeplane/plane/pull/1970))
- Optimized the `n+1` for project members. ([#1975](https://github.com/makeplane/plane/pull/1975))
- Pending member invites have been reordered to appear at the start of the members' page. ([#1978](https://github.com/makeplane/plane/pull/1978))
- Introduced `workspaceSlug` to the endpoint for my profile activity. ([#1983](https://github.com/makeplane/plane/pull/1983))
- Repositioned the activity loader for `My Issues`. ([#1983](https://github.com/makeplane/plane/pull/1983))
- Omit the display of completion percentage if the user does not have any assigned issues. ([#1984](https://github.com/makeplane/plane/pull/1984))
- `Favorite` project is generated only when the plus icon within the `favorites` section of the sidebar is utilized. ([#1993](https://github.com/makeplane/plane/pull/1993))
- Optimized the n+1 for issue history endpoint for comment reactions. ([#1994](https://github.com/makeplane/plane/pull/1994))
- Optimized the n+1 for issue automation tasks for auto closing and auto archiving issues.([#1994](https://github.com/makeplane/plane/pull/1994))
- Resolved the problem related to issue exports in self-hosted instances. ([#1996](https://github.com/makeplane/plane/pull/1996))
- Implemented responsiveness for Cycles and Modules Cards view. ([#1997](https://github.com/makeplane/plane/pull/1997))
- Modified the "Next Link" tag to an anchor tag for exporting downloads. ([#2000](https://github.com/makeplane/plane/pull/2000))
- Tiptap editor export issues have been fixed. ([#2001](https://github.com/makeplane/plane/pull/2001))
- Corrected the Issue creation statistics/counts in the user profile. ([#2016](https://github.com/makeplane/plane/pull/2016))
- Rectified the project progress calculation logic in the Profile Sidebar of User Profile. ([#2018](https://github.com/makeplane/plane/pull/2018))
- Resolved the empty state issue in Kanban layout for Subscribed issues in User Profile. ([#2018](https://github.com/makeplane/plane/pull/2018))
- Addressed the Group-By Property persistence issue when changing the layout. ([#2018](https://github.com/makeplane/plane/pull/2018))
- The issue where the subscribed issues in `My Issues` were not rendering has been fixed. ([#2037](https://github.com/makeplane/plane/pull/2037))
- Fixed issue with display names in Active Cycles. ([#2041](https://github.com/makeplane/plane/pull/2041))
- Fixed the mutation issue in notification counts. ([#2053](https://github.com/makeplane/plane/pull/2053))
- Resolved the issue related to cycle and module selection within the issue details sidebar. ([#2056](https://github.com/makeplane/plane/pull/2056))
- Numerous bugs in the platform have been resolved, resulting in a smoother experience. ([#2036](https://github.com/makeplane/plane/pull/2036), [#2072](https://github.com/makeplane/plane/pull/2072), [#2080](https://github.com/makeplane/plane/pull/2080), [#2082](https://github.com/makeplane/plane/pull/2082), [#2088](https://github.com/makeplane/plane/pull/2088), [#2094](https://github.com/makeplane/plane/pull/2094), [#2102](https://github.com/makeplane/plane/pull/2102), [#2105](https://github.com/makeplane/plane/pull/2105), [#2111](https://github.com/makeplane/plane/pull/2111), [#2114](https://github.com/makeplane/plane/pull/2114), [#2117](https://github.com/makeplane/plane/pull/2117), [#2118](https://github.com/makeplane/plane/pull/2118))

Thanks to all our contributors - [@sriramveeraghanta](https://github.com/sriramveeraghanta), [@pablohashescobar](https://github.com/pablohashescobar), [@aaryan610](https://github.com/aaryan610), [@dakshesh14](https://github.com/dakshesh14), [@NarayanBavisetti](https://github.com/NarayanBavisetti), [@Kritikkkaaa](https://github.com/Kritikkkaaa), [@srinivaspendem](https://github.com/srinivaspendem), [@rhea0110](https://github.com/rhea0110), [@henit-chobisa](https://github.com/henit-chobisa), [@Palanikannan1437](https://github.com/Palanikannan1437), [@vamsi](https://github.com/vamsi), and [@vihar](https://github.com/vihar).

<hr /><em>This discussion was created from the release <a href='https://github.com/makeplane/plane/releases/tag/v0.12-dev'>v0.12-dev</a>.</em>

**Comment by TheZoker** [comment]
Nice update! Is there any reason, the latest updates are not available on docker hub on release?
